question-service/index.js

```diff
@@ -31,7 +31,7 @@ router.get("/level/:difficulty/:questionNumber", getQuestionByDifficulty);
 // To disable checking by cors on using the same port
 app.use("/api/question", router).all((_, res) => {
   res.setHeader("content-type", "application/json");
```

Comment:
Same here regarding the need to set headers manually

Reply:
Question will not load if this header is removed, so I will keep it. I think it's probably because this `setHeader` is specifically for response header while `cors` is for request header, I might be wrong.

Reply:
Same here, just keep `content-type` and remove `Access-Control-Allow-Origin`.

Not too sure about whether it's for request or response either.

Reply:
Similarly, this works perfectly as well! Thanks for spotting this!